Maquetado-Suscribe

Emiliano-dev25052 opened the pull request:
Se sube maquetado de suscribe #11 
![image](https://github.com/user-attachments/assets/3ae73d17-f076-4a35-8470-07df3054aa1e)

- [x] Se crea maquetando suscribe
- [x] Se pone de fondo imagen respectiva

Emiliano-dev25052 committed:
Maquetado-Suscribe

Se sube maquetado de suscribe

JJS74 commented on css/travel.css:
```diff
@@ -663,3 +663,133 @@ gap: 141px;
+.btn_green4:hover {
+    background-color: #fff;
+    color: #00a651;
+}
```
El icono rojo significa que falta un espacio al final?

MacRoberto replied:
Se indicó en clase este mismo error con el ing. Pech, que no debemos dejar estos espacios al final

Emiliano-dev25052 replied:
Github te marca error cuando tienes espacio despues del cierre de tu última llave de codigo. ejemplo 795 } ya no debe ir espacio o salto de linea, osea que ya no debe mostrarte la línea 796 si esta bien en este caso

JJS74 reviewed (approved):
LGTM

MacRoberto commented on css/travel.css:
```diff
@@ -663,3 +663,133 @@ gap: 141px;
+.btn_green4:hover {
+    background-color: #fff;
+    color: #00a651;
+}
```
Elimina espacio al final de tu código

Emiliano-dev25052 replied:
![image](https://github.com/user-attachments/assets/40aa9928-8de8-440d-979f-ed34e5484715)
Se mantiene eliminado el espacio final

Fernando7Josue commented on css/travel.css:
```diff
@@ -663,3 +663,133 @@ gap: 141px;
+  color: #00a651;
+}
+
+.suscribe {
```
Se repite tu codigo css. favor de checarlo

Emiliano-dev25052 replied:
![image](https://github.com/user-attachments/assets/fef27224-ac41-4d8c-8111-c8eea907ad8c)
Se resuelve duplicado de css

Emiliano-dev25052 committed:
Arreglo Css Suscribe

Se elimino el duplicado de css que hubo dentro del codigo y se elimina el espacio final de css

LuisAmir04 reviewed (approved):
Si se elimino el duplicado del css deberia de estar correcto

Fernando7Josue reviewed (approved):
LGTM

MacRoberto committed:
se elimina espacio al final

MacRoberto committed:
Espacio final css

MacRoberto commented:
Se corrigen espacios al final que causaban error
